src/core/gui/PdfFloatingToolbox.h

```diff
@@ -9,6 +9,8 @@
 
 #include "control/tools/PdfElemSelection.h"  // for PdfElemSelection
 #include "pdf/base/XojPdfPage.h"             // for XojPdfPageSelectionStyle
+#include "util/raii/CLibrariesSPtr.h"
+#include "util/raii/GObjectSPtr.h"  // for GObjectSPtr
```

Comment:
```suggestion
#include "util/raii/GObjectSPtr.h"  // for GObjectSPtr
```
You don't need to include the first one.